wgengine/monitor/monitor_windows.go

```diff
@@ -7,273 +7,159 @@ package monitor
 import (
 	"context"
 	"errors"
-	"fmt"
-	"runtime"
+	"golang.zx2c4.com/wireguard/windows/tunnel/winipcfg"
```

Comment:
split std and third-party packages like goimports does (you should use goimports probably)

Reply:
done